src/main/java/in/koreatech/koin/admin/land/dto/AdminLandsRequest.java

```diff
@@ -0,0 +1,133 @@
+package in.koreatech.koin.admin.land.dto;
+
+
+import java.util.List;
+
+import static com.fasterxml.jackson.databind.PropertyNamingStrategies.SnakeCaseStrategy;
+import com.fasterxml.jackson.databind.annotation.JsonNaming;
+
+import in.koreatech.koin.domain.land.model.Land;
+import io.swagger.v3.oas.annotations.media.Schema;
+
+import jakarta.validation.constraints.NotNull;
+import jakarta.validation.constraints.Pattern;
+import jakarta.validation.constraints.PositiveOrZero;
+import jakarta.validation.constraints.Size;
+
+@JsonNaming(SnakeCaseStrategy.class)
+public record AdminLandsRequest(
+    @Schema(description = "이름 - not null - 최대 255자", example = "금실타운", required = true)
```

Comment:
###R
required 옵션은 Deprecated 되었습니다. requiredMode 로 수정해주세요
```suggestion
    @Schema(description = "이름 - not null - 최대 255자", example = "금실타운", requiredMode = Required)
```

Reply:
이것도 옛날 레가시를 따라가다보니 맞추지 못했네요..
계속 레가시 코드에 벗어나지 못해서 지적이 나오는군요